src/components/Card/Card.tsx

```diff
@@ -27,20 +28,20 @@ const Card = ({
               text="Open"
               variant="secondary"
               onClick={() => {
-                router.push(`/ofd/id/${data.id}`);
+                router.push(`/ofd/id/${id}`);
               }}
             ></tds-button>
           </div>
         </div>
-        <h3 className={styles.card__top__name}>{data.name}</h3>
-        <p className={styles.card__top__description}>{data.description}</p>
+        <h3 className={styles.card__top__name}>{name.split("/").pop()}</h3>
+        <p className={styles.card__top__description}>{description}</p>
       </div>
       <div className={styles.card__bottom}>
         <dl className={styles.card__data}>
           <dt className={styles.card__data__key}>State</dt>
-          <dd>{"Saved"}</dd>
+          <dd>{isDraft ? "Draft" : "Saved"}</dd>
           <dt className={styles.card__data__key}>Last modified</dt>
-          <dd>{data.updatedAt.slice(0, 10)}</dd>
+          <dd>{convertToLocalTime(updatedAt)}</dd>
```

Comment:
Good, it's nice to add the time to the date.

I do believe we can format it better though, for example, it could look like this right now;
9 juni 2024 kl. 5:40:18 em

Should we maybe remove the seconds..?

Reply:
Right now its like June 12, 2024 at 11:15:01 AM, which converts the GMT time to local time, we can keep the seconds in my opinion, let's get UX opinion on this and rework later?